lib/charms/loki_k8s/v0/loki_push_api.py

```diff
@@ -1553,9 +1565,14 @@ def __init__(
         )
         super().__init__(charm, relation_name, alert_rules_path, recursive)
         events = self._charm.on[relation_name]
+
         self.framework.observe(self._charm.on.upgrade_charm, self._on_logging_relation_changed)
         self.framework.observe(events.relation_changed, self._on_logging_relation_changed)
-        self.framework.observe(events.relation_departed, self._on_logging_relation_departed)
+
+        # This will keep track of broken relations so that we can ignore the endpoints
+        # we find in their application data bags
+        self._ignored_relations = []  # type: list
```

Comment:
Afaik, relation-broken is only seen by the departing unit - the remaining unit does not see this event.
Is this correct?